README.md

```diff
@@ -66,5 +66,15 @@ Smaller bundles FTW!!!
 
 Let us know if you're using `preact-material-components` and love them. Send us a PR and add your projects and link right here.
 
+## Important!! less Users
```

Comment:
Please change the heading to FAQ
add a section
>### Error while using `preact-material-components` with `less`
and then add the answer...

Thanks a lot for fixing this, its a huge contribution for us

Reply:
"Important!!" is maybe a little too dramatic for documentaion 😛